static/src/js/actions/preferences.js

```diff
@@ -39,16 +42,17 @@ export const updatePreferences = data => (dispatch, getState) => {
 
       dispatch(updateAuthTokenFromHeaders(headers))
       dispatch(setPreferences(preferences))
-
       dispatch(setIsSubmitting(false))
+      dispatch(pushSuccessNotification(PREFERENCES_SAVED_NOTIFICATION))
```

Comment:
Given that we're removing the reducers, this can be simplified by just calling
```
    addToast('Preferences saved!', {
      appearance: 'success',
      autoDismiss: true
    })
``` 

(additionally, please see my comments regarding the addToast action)